tests/test_kinesisvideo/test_server.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import unicode_literals
+
+import sure  # noqa
+
+import moto.server as server
+from moto import mock_kinesisvideo
+
+"""
+Test the different server responses
+"""
+
+
+@mock_kinesisvideo
+def test_kinesisvideo_list():
+    backend = server.create_backend_app("kinesisvideo")
+    test_client = backend.test_client()
+    # do test
```

Comment:
Is this test necessary, if it's already covered by the boto3 test?

Reply:
It's better to have test for moto_server. I added assert of response code  to make sure that the server is up.
https://github.com/spulec/moto/pull/3271/commits/a7b46c681f66c9a012716f14d6a07a689e1c3fc1